bin/__tests__/jscodeshift-test.js

```diff
@@ -20,6 +20,7 @@ const path = require('path');
 const temp = require('temp');
 const mkdirp = require('mkdirp');
 const testUtils = require('../../utils/testUtils');
+const { chdir } = require('process');
```

Comment:
Nit: Remove spaces - ~~Facebook~~ Meta style has no spaces around destructured variables

```
const {chdir} = require('process');
```

I guess we should configure Prettier for jscodeshift one day.

Reply:
@Daniel15 fixed.